File: src/PolarConverter.JSWeb/App/Models/PolarFile.ts
/// <reference path="../_all.ts" />
module PolarConverter {
    "use strict";
    export interface PolarFile extends PolarConverter.File {
        fileType: PolarConverter.fileType;
        sport: PolarConverter.sportEnum;
        checked: boolean;
        gpxFile: PolarConverter.GpxFile;
    }
}

File: src/PolarConverter.JSWeb/Models/PolarFile.cs
namespace PolarConverter.JSWeb.Models
{
    public class PolarFile: File
    {
        public FileType Type { get; set; }
        public Sport Sport { get; set; }
        public bool Checked { get; set; }
        public GpxFile GpxFile { get; set; }
    }
}